cli/ts/setVerifyingKeys.ts

```diff
@@ -95,6 +95,16 @@ const configureSubparser = (subparsers: any) => {
             help: 'The .zkey file for the vote tallying circuit. '
         }
     )
+    
+    createParser.addArgument(
+        ['-ss', '--subsidy-zkey'],
+        {
+            action: 'store',
+            type: 'string',
+            required: true,
```

Comment:
It causes [`cli/test1.sh`](https://github.com/appliedzkp/maci/blob/v1/cli/test1.sh) [fail](https://github.com/gurrpi/maci/runs/6047070045?check_suite_focus=true). As long as I remember from last weekly call, subsidy functionality should be optional.

Reply:
I made the subsidy calculation as optional now. Could you help to add test case for subsidy calculation as well? test.sh is the script to test multiple batches including subsidy calculation. I also updated the outdated test1.sh to test without subsidy case.

Reply:
Sure, filed an issue: https://github.com/appliedzkp/maci/issues/413